roles/openshift_health_checker/openshift_checks/logging/logging.py

```diff
@@ -15,7 +15,7 @@ class LoggingCheck(OpenShiftCheck):
     logging_namespace = "logging"
 
     def is_active(self):
-        logging_deployed = self.get_var("openshift_hosted_logging_deploy", type=bool, default=False)
+        logging_deployed = self.get_var("openshift_hosted_logging_deploy", convert=bool, default=False)
```

Comment:
"type" is nice and intuitive for the most of what we're doing now with values coming out of `task_args`. I don't think pylint complains since it's never used as a name. But I think we may come up with more interesting standardized conversion and validation functions that don't just translate to a type. For instance, byte size converters, or validation that `deployment_type` is one of the known values.

So it made sense to me to give it a more general name that doesn't imply a simple typecasting. I'm not set on `convert` though, suggestions welcome... I could easily see `make`, `transform`, `expect`, `ensure` ... or accepting multiple names (`type`, `convert`, `validate`), to support not having to remember which it is :)